app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Proposal
+  class CollectionDraftProposalsController < CollectionDraftsController
+
+    def publish
```

Comment:
did you see my comment in Ryan's other PR where he overwrote this method?

Reply:
Check publish() in collection_templates_cotroller.rb

Reply:
Raised exception and redirect to normal mmt path